charts/newrelic-logging/README.md

```diff
@@ -2,7 +2,51 @@
 
 ## Chart Details
 
-This chart will deploy the Fluentbit with the New Relic output plugin as a Daemonset.
+New Relic offers a [Fluent Bit](https://fluentbit.io/)  output [plugin](https://github.com/newrelic/newrelic-fluent-bit-output)  to easily forward your logs to [New Relic Logs](https://docs.newrelic.com/docs/logs/new-relic-logs/get-started/introduction-new-relic-logs). This plugin is also provided in a standalone Docker image that can be installed in a [Kubernetes](https://kubernetes.io/)  cluster in the form of a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/), which we refer as the Kubernetes plugin.
```

Comment:
@jsubirat This is very literal nitpicking already, but why do we have two spaces after the links?

Reply:
Because I accidentally left them. :-D Fixing them, thanks!